src/services/NotificationService.ts

```diff
@@ -0,0 +1,49 @@
+// @ts-ignore
+import { NotifyClient } from "notifications-node-client";
+import {TEMPLATE_IDS} from "../utils/Enums";
+import {ITesterDetails} from "../models";
+
+/**
+ * Service class for Certificate Notifications
+ */
+class NotificationService {
+  private readonly notifyClient: NotifyClient;
+
+  constructor(notifyClient: NotifyClient) {
+    this.notifyClient = notifyClient;
+  }
+
+  /**
+   * Send multiple emails based on array of user details
+   * @param userDetails
+   */
+  public sendVisitExpiryNotifications(userDetails: ITesterDetails[]) {
+    userDetails.forEach(async (email) => {
```

Comment:
I think the parameter here is not `email` but `userDetail` and you'd have to pass `userDetail.email`

Reply:
fixed